profile/README.md

```diff
@@ -1,12 +1,39 @@
-## Hi there 👋
+## Welcome to the Autoware Foundation! 👋
+![Autoware stack](https://static.wixstatic.com/media/984e93_552e338be28543c7949717053cc3f11f~mv2.png/v1/crop/x_0,y_1,w_1500,h_879/fill/w_1079,h_633,al_c,usm_0.66_1.00_0.01,enc_auto/Autoware-GFX_edited.png)
 
-<!--
+The Autoware Foundation manages the development of Autoware, the world's leading open-source software project for autonomous driving, and sponsors Autoware-based projects all over the world. Autoware was started in 2015 and is being used by over 100 companies on more than 30 vehicles in over 20 different countries worldwide. 
 
-**Here are some ideas to get you started:**
+### Repository overview
+- [autowarefoundation/autoware](https://github.com/autowarefoundation/autoware)
+  - Meta-repository containing `.repos` files to construct an Autoware workspace.
+  - It is anticipated that this repository will be frequently forked by users, and so it contains minimal information to avoid unnecessary differences.
+- [autowarefoundation/autoware_common](https://github.com/autowarefoundation/autoware_common)
+  - Library/utility type repository containing commonly referenced ROS packages. 
+  - These packages were moved to a separate repository in order to reduce CI execution time
+- [autowarefoundation/autoware.core](https://github.com/autowarefoundation/autoware.core)
+  - Core repository containing high-quality and stable ROS packages for Autonomous Driving.
```

Comment:
```suggestion
  - Contains high-quality and stable ROS packages for Autonomous Driving.
```

Reply:
```suggestion
  - Main repository for high-quality, stable ROS packages for Autonomous Driving.
```
First off, thank you for the suggestion. I thought that using `core` was confusing, but couldn't think of a better alternative this morning - `main` is perfect. 

To explain my suggestion, since these are bullet points explaining each repository, I believe it is safe to omit the article at the start of the sentence.